shakenfist/config.py

```diff
@@ -14,6 +17,22 @@ def get_node_name():
     return socket.getfqdn()
 
 
+def etcd_settings(_settings):
+    try:
+        value = Etcd3Client(
+            host='localhost', port=2379, protocol='http', api_path='/v3beta/').get(
+                '/sf/config', metadata=True)
+        if value is None or len(value) == 0:
+            return {}
```

Comment:
Good to log that DB has no config.

Reply:
Fair enough. Until logging is sorted, the lack of config values should be a useful proxy.